tests/conftest.py

```diff
@@ -172,12 +173,87 @@ def dummy_carbon_data(layer_roles_fixture):
     return data
 
 
+@pytest.fixture
+def dummy_litter_data(layer_roles_fixture):
+    """Creates a dummy litter data object for use in tests."""
+
+    from virtual_rainforest.core.data import Data
+    from virtual_rainforest.core.grid import Grid
+
+    # Setup the data object with four cells.
+    grid = Grid(cell_nx=3, cell_ny=1)
```

Comment:
Would it be worth having `cell_ny > 1` here? I can imagine a case where the code only works with `cell_ny == 1`.

Reply:
It actually should be fine, because the 'grid' is always a one-dimensional array axis indexed by `grid_id`.